tests/zfs-tests/tests/functional/cli_root/zfs_create/zfs_create_002_pos.ksh

```diff
@@ -78,4 +81,15 @@ while (( $j < ${#size[*]} )); do
 	((j = j + 1))
 
 done
+
+typeset -i j=0
+while (( $j < ${#explicit_size_check[*]} )); do
+  propertycheck ${TESTPOOL}/${TESTVOL}${explicit_size_check[j]} \
+    volsize=${expected_rounded_size[j]} || \
+    log_fail "volsize ${size[j]} was not rounded up"
+
+	((j = j + 1))
+
```

Comment:
nit: extra space

Reply:
Fixed, but also fixed a stray space in the function above to be consistent.